source/dub/packagesupplier.d

```diff
@@ -292,17 +292,19 @@ class RegistryPackageSupplier : PackageSupplier {
 
 package abstract class AbstractFallbackPackageSupplier : PackageSupplier
 {
-	protected PackageSupplier m_default, m_fallback;
+	protected PackageSupplier m_default;
+	protected PackageSupplier[] m_fallbacks;
 
-	this(PackageSupplier default_, PackageSupplier fallback)
+	this(PackageSupplier default_, PackageSupplier[] fallbacks)
 	{
 		m_default = default_;
-		m_fallback = fallback;
+		m_fallbacks = fallbacks;
 	}
 
 	override @property string description()
 	{
-		return format("%s (fallback %s)", m_default.description, m_fallback.description);
+		import std.algorithm : map;
+		return format("%s (fallback %s)", m_default.description, m_fallbacks.map!(x => x.description));
```

Comment:
Not a blocker, but this is fairly long and detailed.
```
==== registry at https://code.dlang.org/ (fallback ["registry at http://code.dlang.org/", "registry at https://code-mirror.dlang.io/", "registry at https://code-mirror2.dlang.io/", "registry at https://dub-registry.herokuapp.com/"]) ====
```

Reply:
I know, but the `PackageSupplier` interface currently only supports a `description` attribute. We could add sth. like `location` in a follow-up?